fastly/object_store.go

```diff
@@ -0,0 +1,214 @@
+package fastly
+
+import (
+	"io"
+	"net/http"
+	"strings"
+	"time"
+)
+
+// https://developer.fastly.com/reference/api/object-store/
+
+// TODO(dgryski): Should the store id be `StoreID` or `ID` consistently?
```

Comment:
It's historically been mixed, but I tend to use `ID` and rely on the name of the struct to provide the context for the type of ID. If it's ambiguous or there is more than one type of ID that can be passed to the struct then I'd lean towards a more explicit `StoreID`.

Reply:
Ok, then I can reuse the `ErrMissingID` error instead of creating a new one.  I'll update field to be just `ID`.